ReactAndroid/src/main/java/com/facebook/react/views/text/ReactInlineBorderSpan.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.views.text;
+
+import android.graphics.Paint;
+import android.text.style.LineBackgroundSpan;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.graphics.Color;
+
+/**
+ * We use a custom {@link LineBackgroundSpan}, because RN Text component can't render borders. Details here:
+ * https://github.com/Expensify/App/issues/4733
+ */
+public class ReactInlineBorderSpan implements LineBackgroundSpan, ReactSpan {
+  private final int effectiveStart;
+  private final int effectiveEnd;
+  private final int effectiveFontSize;
+
+  private final int backgroundColor = Color.parseColor("#f6f6f6");
+  private final int borderColor = Color.parseColor("#b3b8bc");
+  private final int borderRadius = 8;
+  private final int borderWidth = 2;
```

Comment:
This isn't a blocker, but I wonder if this and the colours are something we can define at the JReact level? This would make it far simpler to allow users to switch theme

Reply:
Working on this.

Reply:
yep, any thing that is style should come from JReact, thanks for working on it :)
I'm looking at the iOS changes to see if I can identify them all

Reply:
Implemented in both Android and iOS